src/pages/ServiceDetails/MetricsChartBase.tsx

```diff
@@ -1,20 +1,31 @@
 import * as React from 'react';
-import { LineChart } from 'patternfly-react';
+import { LineChart, Icon } from 'patternfly-react';
 import { TimeSeries } from '../../types/Metrics';
+import { style } from 'typestyle';
 
 type MetricsChartBaseProps = {
   familyName: string;
+  onExpandRequested?: () => void;
 };
 
+const expandBlockStyle = style({
+  marginBottom: '-1.5em',
+  zIndex: 1,
```

Comment:
Why do we have to change the z-index?

Reply:
You can see that I have a negative `margin-bottom`. This is to place the _Expand_ link close to the chart (where I think it looks better). But this also means that the c3 control and the _Expand_ link are overlapping. The link is not clickable if the `z-index` is not adjusted.